spot-client/src/spot-tv/ui/components/meeting-frame/ZoomMeetingFrame/ZoomMeetingFrame.js

```diff
@@ -161,6 +172,17 @@ export class ZoomMeetingFrame extends AbstractMeetingFrame {
             type
         });
 
+        if (typeof type === 'object') {
```

Comment:
Can this code go to the AbstractMeetingFrame? It appears to be duplicated between Jitsi and Zoom meeting frames.

Reply:
Can this be addressed @zbettenbuk ?